src/clj/web/auth.clj

```diff
@@ -198,19 +198,28 @@
                         :resetPasswordExpires reset-expires}})
     reset-code))
 
+(defn get-user-of-email
+  "Get the user object of an email address from the db"
+  [db email]
+  (mc/find-one-as-map db "users"
+                      {:email email})
+  )
+)
+
 (defn forgot-password-handler
   [{db :system/db
     email-settings :system/email
     {:keys [email]} :params
     headers         :headers}]
   (if (find-non-banned-user db {:email email})
     (let [code (set-password-reset-code! db email)
+          user (get-user-of-email db email)
           msg (mail/send-message
                 email-settings
                 {:from    "support@jinteki.net"
                  :to      email
                  :subject "Jinteki Password Reset"
-                 :body    (str "You are receiving this because you (or someone else) have requested the reset of the password for your account.\n\n"
+                 :body    (str "You are receiving this because you (or someone else) have requested the reset of the password for your account " (user :username) ".\n\n"
```

Comment:
We probably need to sanitize this somehow. Otherwise we would send user generated text to the email address. Not sure what rules we would want to sanitize against.

Reply:
Do we do any sanitization for usernames already? If so, should be good to go. If not, then hmmm

Reply:
It appears there is no sanitation for usernames. See: [netrunner/src/clj/web/auth.clj:66`](https://github.com/mtgred/netrunner/pull/6254/files#diff-9457c792e3eb8cde9946d207495013396702673433db6ff9d8676e49a7834f4fL66)

But also, I'm pretty sure the email is sent as `plain/text` and is not an HTML email. In order to have an HTML email a different `:content_type` must be specified via `postal` it appears.